### PR TITLE
fix ns log perf problem

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
Issues: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by intellinjun** [comment]
https://inteltf-jenk.sh.intel.com/job/neural_speed_extension/39/

File: neural_speed/models/model_utils/model_utils.h [hunk 1]
```diff
@@ -49,7 +49,15 @@
 #define MODEL_SESSION_MAGIC MODEL_FILE_MAGIC_GGSN
 #define MODEL_SESSION_VERSION 1
 
-int64_t ns_log_level();
+inline int64_t ns_log_level() {
+  static int64_t log_level = -1;
+  if (log_level == -1) {
```
Comment (airMeng) [inline neural_speed/models/model_utils/model_utils.h:54]: meaningless?

Comment (zhenwei-intel) [reply neural_speed/models/model_utils/model_utils.h:54]: static variable log_level, when it is set to 0/1/2, we will ignore env

**Review by a32543254 — APPROVED** [review]
LGTM

File: neural_speed/models/llama/llama.cpp [hunk 2]
```diff
@@ -399,9 +399,9 @@ static bool llama_model_eval_internal(model_context* ctx, const model_input* inp
   ne_build_forward_expand(&gf, inpL);
   ne_graph_compute(ctx0, &gf);
 
-  if (ns_log_level() == 0 || ns_log_level() == 2) {
-    ne_graph_profiling(&gf);
-  }
+  // if (ns_log_level() == 0 || ns_log_level() == 2) {
+  // ne_graph_profiling(&gf);
+  // }
```
Comment (DDEle) [inline neural_speed/models/llama/llama.cpp:404]: Why no profiling for llama?

Comment (zhenwei-intel) [reply neural_speed/models/llama/llama.cpp:404]: https://github.com/intel/neural-speed/pull/81

Comment (zhenwei-intel) [reply neural_speed/models/llama/llama.cpp:404]: fixed in this pr, merged

**Pull request merged**